modules/uninstalling-olm-using-ansible.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * install_config/uninstall-olm-using-ansible.adoc
+
+[id="uninstalling-olm-using-ansible_{context}"]
+= Uninstalling Operator Lifecycle Manager using Ansible
+
+After installing your cluster, you can uninstall the Technology Preview Operator Framework by using he {product-title} `openshift-ansible` installer.
+
+.Prerequisites
+
+- An existing {product-title} 3.11 cluster
+- Access to the cluster using an account with `cluster-admin` permissions
+- Ansible playbooks provided by the latest `openshift-ansible` installer
+
+.Procedure
+
+. Add the following variables to your `config.yml` playbook:
++
+[source,yaml]
+----
+operator_lifecycle_manager_install=false
+operator_lifecycle_manager_remove=true
+----
+
+. Change to the playbook directory:
++
+[source,terminal]
```

Comment:
This is another difference - we don't specify `source,terminal` for commands in 3.11 docs. We just have nothing above them.

Reply:
Addressed